src/lib/select/select.spec.ts

```diff
@@ -1362,7 +1369,8 @@ describe('MdSelect', () => {
   selector: 'basic-select',
   template: `
     <div [style.height.px]="heightAbove"></div>
-    <md-select placeholder="Food" [formControl]="control" [required]="isRequired">
+    <md-select placeholder="Food" [formControl]="control" [required]="isRequired"
+      [tabIndex]="tabIndexOverride">
```

Comment:
It would be nice to use `tabindex` instead of `tabIndex`. That way users can set tabindex as they normally would without having to learn a different API for the select.

```html
<md-select tabindex="6">
</md-select>
```